ibis/expr/types/relations.py

```diff
@@ -1334,6 +1334,66 @@ def sql(self, query: str) -> ir.Table:
         )
         return op.to_expr()
 
+    def histogram(
+        self,
+        column: str,
+        nbins: int | None = None,
+        binwidth: float | None = None,
+        base: float | None = None,
+        aux_hash: str | None = None,
+        eps: float = 1e-13,
+    ):
+        """Compute a histogram with fixed width bins.
+
+        Parameters
+        ----------
+        column
+            Column to bucket
+        nbins
+            If supplied, will be used to compute the binwidth
+        binwidth
+            If not supplied, computed from the data (actual max and min values)
+        base
+            Histogram base
+        aux_hash
+            Auxiliary hash value to add to bucket names
+        eps
+            Allowed floating point epsilon for histogram base
+
+        Returns
+        -------
+        Table
+            New table expression with a bucketed column expression
+        """
```

Comment:
Would you mind expanding the docstring here a bit? Questions I have reading this:

- I can't picture the output schema of this operation. Maybe an example would help clarify?
- What does "Histogram base" mean?

I recognize that this was just copy-pasted from the column method, so if you don't want to tackle this now that's fine.

Reply:
Done, thanks for calling this out!